crates/livesplit-hotkey/src/macos/key_code.rs

```diff
@@ -0,0 +1,262 @@
+    B,
+    C,
+    D,
+    E,
+    F,
+    G,
+    H,
+    I,
+    J,
+    K,
+    L,
+    M,
+    N,
+    O,
+    P,
+    Q,
+    R,
+    S,
+    T,
+    U,
+    V,
+    W,
+    X,
+    Y,
+    Z,
+
+    F1,
+    F2,
+    F3,
+    F4,
+    F5,
+    F6,
+    F7,
+    F8,
+    F9,
+    F10,
+    F11,
+    F12,
+    F13,
+    F14,
+    F15,
+    F16,
+    F17,
+    F18,
+    F19,
+    F20,
+
+    Numpad0,
+    Numpad1,
+    Numpad2,
+    Numpad3,
+    Numpad4,
+    Numpad5,
+    Numpad6,
+    Numpad7,
+    Numpad8,
+    Numpad9,
+    NumpadClear,
+    NumpadDecimal,
+    NumpadDivide,
+    NumpadEnter,
+    NumpadEquals,
+    NumpadMinus,
+    NumpadMultiply,
+    NumpadPlus,
+
+    Backslash,
+    Comma,
+    Equal,
+    Grave,
+    LeftBracket,
+    Minus,
+    Period,
+    Quote,
+    RightBracket,
+    Semicolon,
+    Slash,
+    CapsLock,
+    Command,
+    Control,
+    Delete,
+    DownArrow,
+    End,
+    Escape,
+    ForwardDelete,
+    Function,
+    Help,
+    Home,
+    IsoSection,
+    JisEisu,
+    JisKana,
+    JisKeypadComma,
+    JisUnderscore,
+    JisYen,
+    LeftArrow,
+    Mute,
+    Option,
+    PageDown,
+    PageUp,
+    Return,
+    RightArrow,
+    RightCommand,
+    RightControl,
+    RightOption,
+    RightShift,
+    Shift,
+    Space,
+    Tab,
+    UpArrow,
+    VolumeDown,
+    VolumeUp,
+}
+
+impl FromStr for KeyCode {
+    type Err = ();
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        use self::KeyCode::*;
+        Ok(match s {
+            "Digit0" => Digit0,
+            "Digit1" => Digit1,
+            "Digit2" => Digit2,
+            "Digit3" => Digit3,
+            "Digit4" => Digit4,
+            "Digit5" => Digit5,
+            "Digit6" => Digit6,
+            "Digit7" => Digit7,
+            "Digit8" => Digit8,
+            "Digit9" => Digit9,
+
+            "A" => A,
+            "B" => B,
+            "C" => C,
+            "D" => D,
+            "E" => E,
+            "F" => F,
+            "G" => G,
+            "H" => H,
+            "I" => I,
+            "J" => J,
+            "K" => K,
+            "L" => L,
+            "M" => M,
+            "N" => N,
+            "O" => O,
+            "P" => P,
+            "Q" => Q,
+            "R" => R,
+            "S" => S,
+            "T" => T,
+            "U" => U,
+            "V" => V,
+            "W" => W,
+            "X" => X,
+            "Y" => Y,
+            "Z" => Z,
+
+            "F1" => F1,
+            "F2" => F2,
+            "F3" => F3,
+            "F4" => F4,
+            "F5" => F5,
+            "F6" => F6,
+            "F7" => F7,
+            "F8" => F8,
+            "F9" => F9,
+            "F10" => F10,
+            "F11" => F11,
+            "F12" => F12,
+            "F13" => F13,
+            "F14" => F14,
+            "F15" => F15,
+            "F16" => F16,
+            "F17" => F17,
+            "F18" => F18,
+            "F19" => F19,
+            "F20" => F20,
+
+            "Numpad0" => Numpad0,
```

Comment:
Is the goal to have these strings be consistent across operating systems?

Reply:
Yeah, otherwise using the library (like us setting NumPad* to be the default for splitting / resetting / ...) is just unnecessarily painful.